### ISSUE TITLE
Binance API hacked

### ISSUE THREAD

**Issue by waaadim** [jaggedsoft/node-binance-api#647]
The attack took place on April 18'th, withdrawals were disabled so the attacker just made transactions to drain the funds.
has anyone else noticed anything?
have not shared my key with anyone

**Comment by OKCOOL1233** [comment]
is this the only place you've used those keys?

**Comment by waaadim** [comment]
yes, the only thing that comes to mind is - I did share my old/previous key with someone, but I generated new ones once I figured out my mistake.
oh, also it had no IP restrictions
